next/docs/events/pag2025.mdx

```diff
@@ -0,0 +1,31 @@
+---
+conference: "PAG 32"
+description: "The Plant and Animal Genome Conference (PAG) is designed to provide a forum on recent developments and future plans for plant and animal genome projects worldwide."
+eventType: "Conference"
+featured: true
+location: "The Town & Country Resort, 500 Hotel Cir N, San Diego, CA, USA"
+sessions:
+  [
+    { sessionEnd: "10 Jan 2025 9:00 AM", sessionStart: "10 Jan 2025 5:00 PM" },
```

Comment:
@esthermcdade I think the sessionEnd should be 5pm, and the sessionStart 9am... here and below... 😁